math/mathcore/src/FitResult.cxx

```diff
@@ -185,6 +185,10 @@ void FitResult::FillResult(const std::shared_ptr<ROOT::Math::Minimizer> & min, c
          fChi2 = (*chi2func)(&fParams[0]);
       }
    }
+   else if (fitType == 3) {
+      // case of binned likelihood fits (use Baker-Cousins chi2)
+      fChi2 = 2 * fVal;
```

Comment:
Just a question for my understanding: I understand why you need to do `2 * fVal` for the binned likelihood fit type, but for the unbinned likelihood you still do `fChi2 = fVal` in line 181. Is that intended?

Reply:
This is not really relevant because in this case the chi2 will not be printed by the FitResult.

Reply:
Okay!